docs/administrator/deploy/upgrade.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: Upgrading applications
+---
+
+# Upgrading applications
+Upgrading an application allows for a new version of a released project to be deployed to an existing application. Tasks and process instances that are in progress and based on a previous application version can still be completed, however any new ones started will use the new model definitions. 
+
+Some considerations for upgrading applications to be aware of are that:
+
+* An application can be upgraded to a released project version that is lower than the one currently deployed, however the application version will still increment.   
+* Applications can only be upgraded through the deployment service. This means that if an application was deployed via Helm it cannot be upgraded.
+* [UIs](../../modeling/interfaces.md) will be added and removed between versions based on whether they exist in the project. If only the `type` is changed and the `name` remains the same then the `type` will not be updated between versions. 
+* It is not possible to change an application name when upgrading an application. 
```

Comment:
i am not sure we should add this line, they ui won't allow you to change the app name anyway

Reply:
Good point - I've removed it.